### ISSUE TITLE
Payment Log "order data" breaks backend list

### ISSUE THREAD

**Issue by dathwa** [OFFLINE-GmbH/oc-mall-plugin#655]
Navigate to payment logs `backend/offline/mall/paymentlogs`.
Make the "order data" column visible...

We're sorry, but an unhandled error occurred. Please see the details below.

```
htmlentities() expects parameter 1 to be string, array given
/modules/backend/widgets/Lists.php line 1187
```

**Comment by tobias-kuendig** [comment]
Looks like we want to show array data in the list. Can you figure out which column triggers this error? We might have to add a `d()` around that column as it is done with other fields in the payments log.

**Comment by dathwa** [comment]
Yep. in columns.yaml, type was set to text. you already had a partial defined `$/offline/mall/controllers/paymentlogs/_order_data.htm`
so i updated the definition
```
type: partial
path: $/offline/mall/controllers/paymentlogs/_order_data.htm
```

and this fixed it.

**Comment by tobias-kuendig** [comment]
Fixed in c1a37b0, thanks
